GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -12,4 +12,4 @@
 	<key>ORGANIZATIONNAME</key>
 	<string>Government Digital Services</string>
```

Comment:
Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579135241)